sajibportfolio.html

```diff
@@ -1,273 +1,146 @@
-          </li>
-          <li class="li3">
-            <p class="lp3">javascript</p>
-          </li>
-        </ul>
-        <div class="btl">
-          <button class="btn"><a href="#" class="btt">See Project</a></button>
-        </div>
-
-      </div>
-
-    </section>
-  </section>
-
-  <section id="about_me">
-    <div class="top">
-      <h2 class="pta1">
-        About <br />
-        Myself
-      </h2>
-      <p class="pta2">
-        Hello I’m a data Scientist and software developer! I can help you
-        build a product , feature or website Look through some of my work and
-        experience! If you like what you see and have a project you need
-        coded, don’t hestiate to contact me.
-      </p>
-      <div class="smd">
-        <div class="st">
-          <h2 class="stt">LET’S CONNECT</h2>
-        </div>
-        <ul class="fra">
-          <li class="nb1">
-            <a href="https://github.com/sajibarafatsiddiqui/" target="_blank"><img src="statics/images/github.png"
-                alt="github profilelink" /></a>
-          </li>
-          <li class="nb1">
-            <a href="https://www.linkedin.com/in/sajib-arafat-siddiqui-7aa8b366" target="_blank"><img
-                src="statics/images/Linkedin_icon.png" alt="linkedin profile link" /></a>
-          </li>
-          <li class="nb1">
-            <a href="#" target="_blank"><img src="statics/images/doits.png" alt="angel list link" /></a>
-          </li>
-          <li class="nb1">
-            <a href="#" target="_blank"><img src="statics/images/twitter.png" alt="angel list link" /></a>
-          </li>
-          <li class="nb1">
-            <a href="#" target="_blank"><img src="statics/images/M.png" alt="angel list link" /></a>
-          </li>
-        </ul>
-        <button class="rsm"><a class="brt" target="_blank" href="#">Get my resume</a></button>
-      </div>
-
-    </div>
-
-    <div id="my_skills">
-
-    <ul class="ss">
-      <li>  
-            <div id="spec_name">
-              <p class="lfs">Languages</p>
-              <img class="lfsi" src="statics/images/fleche_dessous.png">
-            </div>
-
-            <ul id="programmes">
-              <li class="pro">
-                <img class="img1" src="statics/images/java.png" alt="javascript image">
-                <p class="lng1">Javascript</p>
-              </li>
-              <li class="pro">
-                <img class="img1" src="statics/images/html.png" alt="html image">
-                <p>HTML</p>
-              </li>
-              <li class="pro">
-                <img class="img1" src="statics/images/css.png" alt="css image">
-                <p>CSS</p>
-              </li>
-            </ul>
-
-          
-       
-      </li>
-      <li>
-        <div id="spec_name">
-          <p class="lfs">Frameworks</p>
+        <div id="frameworks">
+          <p>Frameworks</p>
           <div><img src="statics/images/select.png" alt="select"></div>
```

Comment:
- [ ] I can see that you did some effort to make your disign match the Figma design but I just noticed that you still need to improve on this part, as required for this project your design should sticks to the design templates in Figma as much as possible, this includes the font, colors, images and spacing (margin/padding), kindly improve your design to look more attractive to recruiters so that they can reach you out :wink:  

Your design 
![Screenshot from 2022-12-07 02-04-13](https://user-images.githubusercontent.com/88894525/206052551-57399d9c-71fe-46e2-8242-035507397456.png)

Figma design 
![fig,a](https://user-images.githubusercontent.com/88894525/206052596-8cceaa79-4499-4bdc-9581-a3811db307fe.png)